Rakefile

```diff
@@ -12,7 +12,10 @@ rescue LoadError
   # should only be here when gem group development and test aren't installed
 end
 
-task default: [:javascript_tests, :spec, :rubocop]
+# remove default rspec task
+task(:default).clear
```

Comment:
was this causing issues?

Reply:
Yes, if you don't clear it, it just appends to it rather than overwriting it.